e2e/utils/zetacore.go

```diff
@@ -314,19 +314,20 @@ func WaitForBlockHeight(
 
 	var currentHeight int64
 	for i := 0; currentHeight < desiredHeight; i++ {
+		time.Sleep(1 * time.Second)
 		s, err := rpc.Status(ctx)
 		if err != nil {
-			return errors.Wrap(err, "unable to get status")
+			continue
 		}
-
 		currentHeight = s.SyncInfo.LatestBlockHeight
 
-		time.Sleep(1 * time.Second)
-
 		// prevent spamming logs
 		if i%10 == 0 {
```

Comment:
NIT: Maybe we should make this`6` so that we check once every block. 
We can increase the condition for max value of i , if we do so